lib/streamlit/runtime/scriptrunner/script_runner.py

```diff
@@ -322,7 +322,10 @@ def _get_script_run_ctx(self) -> ScriptRunContext:
             If there is no ScriptRunContext for the current thread.
 
         """
-        assert self._is_in_script_thread()
+        if not self._is_in_script_thread():
+            raise RuntimeError(
+                "get_script_run_ctx must be called from the script thread."
```

Comment:
Nit: Perhaps `ScriptRunner._get_script_run_ctx must be called from the script thread.` for consistency with below